.github/ISSUE_TEMPLATE/user-story.yml

```diff
@@ -0,0 +1,62 @@
+name: User Story
+description: Capture actionable sprint work
```

Comment:
@PaulKuykendall may have preferences for what this should say. 

This is the text that will appear under the "User story" option when selecting the type of issue you want to create on the https://github.com/cisagov/getgov/issues/new/choose page